File: launchschool_exercises/210_small_problems/easy1/tip-calculator.js
var bill = parseFloat(prompt('What is the bill? '));
var tip = parseFloat(prompt('What is the tip percentage? '));

console.log('The tip is $' + (bill * (tip * 0.01)));
console.log('The total is $' + (bill + (bill * (tip * 0.01))));

File: launchschool_exercises/109_small_problems/easy2/tip_calculator.rb
print 'What is the bill? '
bill = gets.chomp.to_f
print 'What is the tip percentage? '
percentage = gets.chomp.to_f

tip = (bill * (percentage / 100.0))
total = bill + tip

puts "The tip is $#{sprintf('%.2f', tip)}"
puts "The total is $#{sprintf('%.2f', total)}"
